pkgs/build-support/fetchnextcloudapp/default.nix

```diff
@@ -4,7 +4,7 @@
 , sha256 ? ""
 , appName ? null
 , appVersion ? null
-, license
+, licenses
```

Comment:
This is a breaking change to the API.

Documenting that and keeping `license` for a bit and throwing a better error if it's not null should be OK here.